web/sites/default/example.settings.local.php

```diff
@@ -1,10 +1,10 @@
 <?php
 /**
  * @file
- * amazee.io Drupal 8 example settings.local.php file
+ * Local settings file example.
  *
- * This file will not be included and is just an example file.
- * If you would like to use this file, copy it to the name 'settings.local.php' (this file will be exluded from Git)
+ * This file will not be included and is just an example file. Copy this to
```

Comment:
`Copy this file to enable` or `Copy it to enable`

Reply:
> Do I understand it correctly that the whitelisting is now managed through a metapackage `GovCMS/whitelist`, which is included by default (it would be great if it is as this is a very clean way to do so)? This part is a bit confusing from the point of permissions and file locks: who can edit which file? Can agency developers edit `composer.json` in SaaS? What about SaaS+?

My initial effort is to "make a paas that could be used for saas" so some of these distinctions are not set in stone yet. I expect it will be something like:

- saas - all files locked
- saas+ - can change some files like composer.json, but some things are still restricted (eg only use satis.govcms.gov.au)
- paas - can change all files but make the DX experience optimal, and document recommendations, to keep the developers mostly using the tools as-is.